src/tokenizers.js

```diff
@@ -927,6 +1491,10 @@ class PreTrainedTokenizer extends Callable {
         return new this(tokenizerJSON, tokenizerConfig);
     }
 
+    /**
+     * @param {any} inputs
+     * @return {any}
+     */
```

Comment:
Merely to pass-through the type information so that it is not lost:

```suggestion
    /**
     * @param {T} inputs
     * @template T
     * @return {T}
     */
```